### ISSUE TITLE
Newly added secondary contact field doesn't have focus when using keyboard navigation [2]

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-cif#880]
Describe the Bug:

Probability (how likely the bug is to happen, scored from 1-5): 1
(For example, probability of 5 is something like "it happens to all users every time they log in."
Probability of 1 "only happens to certain users when a really specific and unlikely path is followed.")

Effect (how bad the bug is when it does happen, scored from 1-5): 1
(For example, effect of 5 is "the entire app crashes and makes it unusable for all users" or "the bug causes the wrong data to be saved, with critical information (e.g. payment) being affected."
Effect of 1 is "It makes some styling look a little bit weird.")

Steps to reproduce the behaviour:
- in localhost
- use keyboard navigation to navigate to the add secondary contact button and hit enter to click it
- there's no focus on the secondary contact dropdown so you have to tab through half the page to get back to it

Screenshots:
https://www.loom.com/share/1f4c44010814487d80982d00c7621dd1

Additional information:

- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]
- Additional context

**Comment by pbastia** [comment]
@nanyangpro @suhafa we would need to determine if this is a behaviour we would like, and if it's something that is recommended in terms of accessibility and user interaction:
- moving the focus automatically to a different control
- is the behaviour different when using the keyboard or the mouse?
